### PR TITLE
Adds supports for WikibaseDataModel 5 and 6

### PR DESCRIPTION
**Pull request by Tpt**

### CODE REVIEW

**Comment by thiemowmde** [comment]
We worked hard to have a very accurate change log in https://github.com/wmde/WikibaseDataModel/blob/master/RELEASE-NOTES.md. Please make sure you are not using any of the functionality mentioned as a breaking change in 5.0 and 6.0. Being cross-compatible with all versions including 4.0 is fine then.

File: src/DataModel/SerializedEntity.php [hunk 1]
```diff
@@ -77,4 +77,18 @@ public function isEmpty() {
 	public function getSerialization() {
 		return $this->serialization;
 	}
+
+	/**
+	 * @see Comparable::equals
+	 */
+	public function equals( $target ) {
+		$target instanceof SerializedEntity && $this->serialization == $target->serialization;
```
Comment (thiemowmde) [inline src/DataModel/SerializedEntity.php:85]: Should be `===`.

Comment (Tpt) [inline src/DataModel/SerializedEntity.php:85]: I'm not sure we want to enforce array order. So, I would prefer stay with "==".

Comment (thiemowmde) [inline src/DataModel/SerializedEntity.php:85]: Oh, its an array. Never mind then.

**Pull request merged**